tasks/main.yml

```diff
@@ -2,6 +2,17 @@
 # Include variables and define needed variables.
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
+  when: ansible_os_family  != "RedHat"
+
+# Include RedHat 6 variables (mysql)
+- name: Include OS-specific variables.
+  include_vars: "{{ ansible_os_family }}-{{ ansible_lsb.major_release }}.yml"
```

Comment:
If `ansible_lsb.major_release` just outputs `6` or `7`, no need for two tasks, we can just do one task without a `when` condition.